release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1201,6 +1201,132 @@ For more information about the unsupported, community-maintained, version of the
 [id="ocp-release-note-cloud-compute-bug-fixes_{context}"]
 ==== Cloud Compute
 
+* When upgrading {gcp-short} clusters that use a boot disk that is not compatible with UEFI, you cannot enable shielded VM support. 
+Previously, this prevented the creation of new machines. 
+With this release, disks with known UEFI incompatiblity have shielded VM support disabled. 
+This primarily affects customers upgrading from {product-title} version 4.12 to 4.13 using the {gcp-short} marketplace images. 
+(link:https://issues.redhat.com/browse/OCPBUGS-17079[OCPBUGS-17079])
+
+* Previously, virtual machines (VMs) in a cluster that ran on {azure-short} failed because the attached network interface controller (NIC) was in a `ProvisioningFailed` state. 
+With this release, the Machine API controller checks the provisioning status of a NIC and refreshes the VMs on a regular basis to prevent this issue. 
+(link:https://issues.redhat.com/browse/OCPBUGS-31515[OCPBUGS-31515])
+
+* Previously, the certificate signing request (CSR) approver included certificates from other systems within its calculations for whether it was overwhelmed and should stop approving certificates. 
```

Comment:
IDK why these longer bugs start so many newlines? It looks fine in the preview, but you might want to check your editor just in case. (Might just be this PR preview for me, who knows?)

Reply:
I have taken to breaking lines at the end of a sentence for easier diffs :)